### PR TITLE
Use github gist for bug report. Closes #183

### LINKED ISSUE #183: The "Create GitHub issue" in the exception dialog is broken
The option to create a GitHub issue in the unhandled exception dialog is broken and has been for a while. Currently, it posts the stack trace to pastebin.com (which requires an API key). This API key is not in source control, and therefore posting the stack trace won't work.

What we should do instead is to create an anonymous Gist and use the link to that Gist in the issue body.

### PR DESCRIPTION
**Pull request by DoumanAsh**
Don't have build system to verify though...

Reference https://developer.github.com/v3/gists/#create-a-gist

### CODE REVIEW

**Comment by vktr** [comment]
Awesome - let's see if the AppVeyor build passes and then I'll have a look to verify :smile:

**Comment by DoumanAsh** [comment]
Lol i made really stupid mistake. re-upload

**Comment by DoumanAsh** [comment]
Opps... my vim auto-cleans  unnecessary white spaces....

**Comment by DoumanAsh** [comment]
@vktr All looks ok, please check code for anything wrong...
Let me know if i should revert my vim auto-cleaning whitespaces

**Comment by vktr** [comment]
Looks OK, I haven't tried yet but could you revert the changes to `picojson.hpp`? Perhaps put the macro inside the `unhandled_exception_controller.cpp` file?

**Comment by DoumanAsh** [comment]
Ok, done

**Comment by vktr** [comment]
What's the change in `.gitignore`?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ tools/Cake.CMake
 tools/nuget.exe
 tools/PicoTorrent.Libs
 tools/WiX.Toolset
+tags
```
Comment (vktr) [inline .gitignore]: If this is unnecessary it should be reverted.

**Comment by DoumanAsh** [comment]
I added to ignore ctags generated file.
I suppose it is rare to see for VS project, but i generally avoid using IDE...

Well if you mind i can remove it too.

**Comment by vktr** [comment]
Great! :smile: I'll test this soon.

**Comment by DoumanAsh** [comment]
@vktr I was doing quick check with python and found mistake. Status is not part of json response, but part of http response :(
Need to correct it

**Comment by vktr** [comment]
Just tested this and it works great! :smile:

**Comment by DoumanAsh** [comment]
Cool, glad to help out a bit.

**Pull request merged**